test_scripts/API/ButtonSubscription/022_SubscribeButton_App1_App2_on_the_OK_button_event_without_AppID.lua

```diff
@@ -0,0 +1,87 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- Another flow for Subscribe on OK button and sending an event notification without App ID
+-- In case:
```

Comment:
Please update description according to script name and sequence.
Also update steps in check list.